Client/Frontend/Home/HomeTabBanner.swift

```diff
@@ -6,53 +6,53 @@ import Storage
 import Shared
 import UIKit
 
-struct DefaultBrowserCardUX {
+struct HomeTabBannerUX {
     static let cardSize = CGSize(width: 360, height: 224)
     static let logoSize = CGSize(width: 64, height: 64)
     static let learnHowButtonSize: CGSize = CGSize(width: 304, height: 44)
 }
```

Comment:
Is this possible to be just be UX inside the HomeTabBanner class? For less pollution.